app/team.go

```diff
@@ -1165,7 +1165,7 @@ func (a *App) RestoreTeam(teamId string) *model.AppError {
 
 func (a *App) GetTeamStats(teamId string) (*model.TeamStats, *model.AppError) {
 	tchan := a.Srv.Store.Team().GetTotalMemberCount(teamId)
-	achan := a.Srv.Store.Team().GetActiveMemberCount(teamId)
+	aCount, aErr := a.Srv.Store.Team().GetActiveMemberCount(teamId)
```

Comment:
you can probably have the variable names as count, err here?

Reply:
Ok